index.bs

```diff
@@ -1508,11 +1514,11 @@ these steps:
 
   1. Perform the <a spec=html>activation notification</a> steps in |global|'s [=Window/browsing context=].
 
-     Note: This lets a website immediately perform operations on the returned handles that
-     might require user activation, such as requesting more permissions.
-
-     Issue(89): Rather than requiring the website to prompt separately for a writable directory,
-     we should provide some kind of API to request a writable directory in one step.
+  1. Let |permissionStatus| be the result of [=requesting file system permission=]
+     given |result| and |options|.{{OpenDirectoryPickerOptions/mode}}.
+     If that throws an exception, [=reject=] |p| with that exception and abort.
```

Comment:
I'm not sure if forwarding the exception here (and rejection with NotAllowedError below) is the right thing to do, and I'm pretty sure it at least doesn't match our current implementation, where I believe (but didn't double check) we always reject with an AbortError instead?